_includes/github.js

```diff
@@ -6,28 +6,45 @@ function GithubRepository(user, repository) {
 }
 
 
-GithubRepository.prototype.getAssetName = function() {
+GithubRepository.prototype.getAssetName = function () {
   var split_path = document.location.pathname.split("/");
   return split_path[split_path.length - 1];
 }
 
-GithubRepository.prototype.redirectToReleaseFile = function(release) {
+GithubRepository.prototype.redirectToReleaseFile = function (release) {
   console.log(release);
   var assetName = this.getAssetName();
-  console.log("Looking for asset: " + assetName)
+
   var downloaded = false;
   var names = [];
-  for (var i = 0; i < release.assets.length; i+=1) {
-    var asset = release.assets[i];
-    names.push(asset.name);
-    if (asset.name == assetName) {
-      downloadAsset(asset.browser_download_url);
+  if(assetName.length > 0){
```

Comment:
As I wrote earlier: I would like to have a check if the index parameter is null.

Reply:
Yuttapongaon29@gmail.com​

Reply:
addressed in https://github.com/chlegou/download_latest/pull/1